app/src/features/Sources/SourceForm.tsx

```diff
@@ -44,7 +44,7 @@ const useStyles = makeStyles((theme) => ({
     minWidth: 150,
   },
   divider: {
-    marginBlock: theme.spacing(5),
+    margin: `${theme.spacing(4)}px ${theme.spacing(3)}px`,
```

Comment:
Does this work instead ?

```suggestion
    margin: theme.spacing(4, 3),
```

Reply:
Yes !